cmd/crio/main.go

```diff
@@ -199,7 +199,11 @@ func main() {
 		},
 		cli.BoolFlag{
 			Name:  "debug",
-			Usage: "enable debug output for logging",
+			Usage: "enable debug output for logging. This is equivalent to -log-level debug",
+		},
+		cli.StringFlag{
+			Name:  "log-level",
+			Usage: "log CRI-O messages above specified level; one of debug, warn, error, fatal or panic",
```

Comment:
Suggest changing "one of debug" to just "debug" to get this string under 80 characters.  As is it's 83